src/atoms/toogle/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+
+import * as Styles from './styles';
+import { ToggleProps } from './types';
+
+const Toggle = ({
+  isLabelVisible = false,
+  disabled = false,
+  checked = true,
+  labelLeft = 'Off',
+  labelRight = 'On',
+  onClick = () => {},
+}: ToggleProps) => {
+  // tracks the active state so it's easier to test
+  const [state, setState] = useState(checked);
+
+  const handleClick = (value: boolean) => {
+    if (disabled) return;
+
+    setState(value);
+    return onClick(value);
+  };
+
+  return (
+    <Styles.Switcher
```

Comment:
```suggestion
    <Styles.Switcher
      className={className}
```

Add `className` to make it easier for anyone who wants to customize some element of the toggle.

Reply:
I will extend the `HTMLDivElement` attributes and give the developer much more flexibility when using the component.